ghc-tags.cabal

```diff
@@ -32,8 +32,9 @@ executable ghc-tags
      build-depends:    ghc, ghc-boot
   else
      build-depends:    ghc-lib == 9.2.*
```

Comment:
This needs to be bumped to 9.4.* for `ghc-tags` to compile with GHC 9.4.

Reply:
Of course it turns out that if this is bumped, then ghc-tags doesn't compile because the GHC API changed :joy_cat: 

So it needs to be adjusted to GHC 9.4.